include/igl/triangle/triexit.cpp

```diff
@@ -0,0 +1,9 @@
+#include <string>
+#include <stdexcept>
+extern "C"
+{
+  void triexit(int status)
+  {
+    throw std::runtime_error("triexit("+std::to_string(status)+")");
```

Comment:
Btw I am not sure what happens when you put `extern "C"` around a code block and then throw a C++ exception inside...

Reply:
This seems to be undefined behavior (albeit it probably works in practice):
https://stackoverflow.com/questions/15845681/can-c-functions-marked-as-extern-c-throw